content/international-candidates.md

```diff
@@ -52,217 +52,162 @@ keywords:
 
-If your application is successful, you can continue to live, work and study in the UK after 30 June 2021. You will not need to use the points-based immigration system explained here.
+Your adviser can help you get a 'statement of comparability' showing [how your overseas qualifications compare to UK qualifications](https://enic.org.uk/Qualifications/SOC/Default.aspx).
 
-### Changes to the UK’s immigration system
+<p class="call-to-action__action">
+  <a href="https://beta-adviser-getintoteaching.education.gov.uk/">Get an <span>adviser</span></a>
+</p>
 
-The points-based immigration system applies to all non-UK citizens arriving in the UK from 1 January 2021, including those from the EEA or Switzerland.
+### Apply for a place
 
-### Irish citizens
+<p class="call-to-action__action">
+  <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find a postgraduate teacher training <span>course</span></a>
+</p>
 
-Irish citizens can continue to enter, work and study in the UK under the [Commmon Travel Area](https://www.gov.uk/government/publications/common-travel-area-guidance/common-travel-area-guidance) as they did previously.
+Or [search for an undergraduate teacher training course](https://digital.ucas.com/search).
 
-## Choosing the correct visa for your job or course
+### What qualifications you can get
 
-For teaching jobs in England, the main visa route is the ‘[skilled worker visa](https://www.gov.uk/skilled-worker-visa)’.
+#### Qualified teacher status (QTS)
 
-For teacher training in England, the main routes are either the ‘[student visa](https://www.gov.uk/student-visa)’ or the ‘[skilled worker visa](https://www.gov.uk/skilled-worker-visa)’.
+QTS allows you to apply for paid teaching jobs in all schools in England.
 
-Depending on your personal circumstances, you may be eligible for [other visas](/international-candidates#other-visas) which allow you to work or study in the UK.
+You need QTS to work in many English schools, but it’s desirable in all schools.
 
-### Teacher training: skilled worker or student visa?
+#### PGCE with QTS
 
-The type of visa you need will depend on whether you are on a salaried or unsalaried teacher training course.
+You can do a course which combines QTS with a Postgraduate Certificate of Education (PGCE). This includes some academic study.
 
-A ‘salaried’ teacher training course means you are paid to work in a school while you train.
+#### PGCEi
 
-Salaried training types include:
+You do not necessarily have to be in England to study with an English institution. You can do an international Postgraduate Certificate of Education (PGCEi) outside the UK.
 
-* school direct (salaried)
-* Teach First
-* postgraduate teaching apprenticeships
+A PGCEi will help you develop your teaching skills and knowledge.
 
-An ‘unsalaried’ teacher training course means you pay a fee.
+However, it does not lead to QTS and will not qualify you to teach in schools in England which require QTS.
```

Comment:
I'm not sure about this but without a comma there's a hint of ambiguity as to whether all schools in England require QTS or some do and you'd only be excluded from them.

```suggestion
However, it does not lead to QTS and will not qualify you to teach in schools in England, which require QTS.
```